lib/mempool/mempool.js

```diff
@@ -1233,6 +1250,16 @@ class Mempool extends EventEmitter {
     if (!proof.isSane())
       throw new VerifyError(proof, 'invalid', 'bad-aidrop-proof', 100);
 
+    if (this.chain.height + 1 >= this.network.goosigStop) {
+      const key = proof.getKey();
+
+      if (!key)
+        throw new VerifyError(proof, 'invalid', 'bad-airdrop-proof', 100);
+
+      if (key.isGoo())
+        throw new VerifyError(proof, 'invalid', 'bad-goosig-disabled', 100);
```

Comment:
I think @pinheadmz raises a good point. This should probably be a ban score of zero as people might make mistakes, and this particular sanity check doesn't require much processing power.

Reply:
Updated for these errors to have a banscore of 0 and added test coverage of those banscores.